packages/zoe/src/zoeService/types.js

```diff
@@ -214,7 +214,7 @@
  * @property {() => Promise<Allocation>} getFinalAllocation
  * return a promise for the final allocation. The promise will resolve after the
  * seat has exited.
- * @property {() => Subscriber<AmountKeywordRecord>} getExitSubscriber returns a subscriber that
+ * @property {() => Subscriber<any>} getExitSubscriber returns a subscriber that
```

Comment:
please no `any`. it's it `void`?

Reply:
According to `contractFacet/types.js`, 
```
/**
 * @typedef {any} Completion
 * Any passable non-thenable. Often an explanatory string.
 */
```
I'll switch to `Completion`.